test/parallel/test-dgram-socket-buffer-size.js

```diff
@@ -72,3 +72,25 @@ const dgram = require('dgram');
     socket.close();
   }));
 }
+
+function checkBufferSizeError(type, size) {
+  const errorObj = {
+    code: 'ERR_SOCKET_BUFFER_SIZE',
+    type: Error,
+    message: new RegExp('^Could not get or set buffer' +
+      ' size: Error: EINVAL: invalid argument,' +
+      ' uv_' + type + '_buffer_size$')
```

Comment:
Nit: I believe a string (instead of a regex) is enough to get exact matching here. You might want to use a template string instead of concatenation, I think we agreed on that preference.

Reply:
I actually used a template for this but the linter complained which is why I changed it. I'll take another look though.

Reply:
We usually use templates, except if the line becomes too long, in that case you can concatenate templates and string literals:

```js
'Could not get or set buffer size: Error: EINVAL: ' +
`invalid argument, uv_${type}_buffer_size`
```

Reply:
Thanks, updated now.